packages/confusing-browser-globals/test.js

```diff
@@ -4,7 +4,7 @@ let assert = require('assert');
 let globals = require('./');
 
 it('should return an Array of globals', function() {
-  assert.strictEqual(globals.constructor, Array);
+  assert.strictEqual(Array.isArray(globals), true);
```

Comment:
Actually, would you mind changing this to use Jest for testing?

We don't use it in integration tests because of some obscure jsdom issue but it would be great to dogfood it in other places.